src/dll.py

```diff
@@ -0,0 +1,112 @@
+"""This is the module for doubly-linked lists methods."""
+# encoding: utf-8
+
+
+class DLLNODE(object):
```

Comment:
This isn't proper naming per Python conventions. It should be `CamelCase` e.g. `DllNode` or `DLLNode`.